update_database_scripts/master_script.sh

```diff
@@ -293,6 +293,9 @@ fi
 if [ $github == "true" ]; then
     operatingSystem=$(printf  $(uname -s))
     if [ $operatingSystem == "Linux" ]; then
+        # git pull before git pushing
+        git pull origin master
```

Comment:
You might also need to consider running a git stash before this. If someone were to make changes to server files, leave, and then this script gets run, there would be problems

Reply:
Or rather, maybe not right here, but right at the beginning of running this script? before the script creates/edits files